tools/topology/m4/tdfb.m4

```diff
@@ -60,6 +60,10 @@ define(`W_TDFB',
 `	bytes ['
 		$6
 `	]'
+`	enum ['
```

Comment:
I think I need to drop this enum section while there's no vanilla kernel that can parse this topology. I'm not sure what I should do. Maybe it's enough to comment out the enum [] part from m4 so it would be easy to restore it for testing (for myself) and patch it back when it becomes supported in kernel.